### PR TITLE
rm conditional texture border clamp;

### PR DESCRIPTION
**Pull request by bjornbytes**
It is core in ES3.2, which is what's required for LÖVR.

### CODE REVIEW

File: LovrApp/Src/LovrApp_NativeActivity.cpp [hunk 1]
```diff
@@ -172,14 +171,10 @@ void BridgeLovrUnpack(ovrRigidBodyPosef &HeadPose, BridgeLovrPose &Pose, BridgeL
 static void EglInitExtensions()
 {
 	const char * allExtensions = (const char *)glGetString( GL_EXTENSIONS );
-	__android_log_print(ANDROID_LOG_DEBUG, "LOVR", "In nativeactivity, version %s, extensions:\n%s\n", glGetString(GL_VERSION), allExtensions);
```
Comment (mcclure) [inline LovrApp/Src/LovrApp_NativeActivity.cpp]: Did you remove this for a reason?

**Comment by mcclure** [comment]
We might be closing off the possibility of supporting certain Gear VR devices by doing this.

Do we want to just go ahead and say that Lovr requires Oculus Go or later (I don't think any of us even know how to test gear vr?)

File: LovrApp/Src/LovrApp_NativeActivity.cpp (hunk 1 above)
Comment (bjornbytes) [reply LovrApp/Src/LovrApp_NativeActivity.cpp]: Ah, no good reason, let me put that back.

**Comment by bjornbytes** [comment]
There is definitely a tradeoff in doing this, I guess right now I see more value in having a smaller surface area to deal with for mobile, but I am open to changing that if needed.  And yes, Gear VR is essentially "unofficially supported" given there is not much/any testing on it.  Though I did briefly check and it seems the last four generations of Samsung phones should do okay with this extension.

**Pull request merged**